frame/bridge/ethereum/issuing/src/lib.rs

```diff
@@ -79,13 +79,15 @@ pub mod pallet {
 
 	#[pallet::config]
 	#[pallet::disable_frame_system_supertrait_check]
-	pub trait Config: dvm_ethereum::Config {
-		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+	pub trait Config: frame_system::Config + darwinia_evm::Config {
```

Comment:
can we not use  darwinia_evm::Config?

Reply:
Yes, we can remove the `darwinia_evm::Config` bind here. However, I don't think this is necessary now, the darwinia_evm config is simple.